cmake/enclave_cmake_wrappers.cmake

```diff
@@ -144,3 +144,14 @@ function (install_enclaves)
     endif ()
   endforeach (target)
 endfunction (install_enclaves)
+
+function (enclave_enable_code_coverage NAME)
+  if (NOT CODE_COVERAGE)
+    return()
+  endif ()
+
+  # Enable code coverage.
+  enclave_compile_options(${NAME} PRIVATE -g -O0 -fprofile-arcs -ftest-coverage)
+  # Link against libgcov.
+  enclave_link_libraries(${NAME} PRIVATE gcov)
```

Comment:
GCC docs say that -fprofile-arcs must also be passed to the linker. Is that equivalent to just linking with gcov?

Reply:
Yes, that's correct (we need to link against our customized libgcov). The `-fprofile-arcs` option to the linker will pick the library installed in the system.